control.go

```diff
@@ -308,10 +325,16 @@ func (c *controlConn) setupConn(conn *Conn) error {
 		return err
 	}
 
-	host = c.session.ring.addOrUpdate(host)
+	var exists bool
+	host, exists = c.session.ring.addOrUpdate(host)
 
 	if c.session.cfg.filterHost(host) {
-		return fmt.Errorf("host was filtered: %v", host.ConnectAddress())
+		return fmt.Errorf("host was filtered: %v (%s)", host.ConnectAddress(), host.HostID())
+	}
+
+	if !exists {
+		c.session.logger.Info("Adding host.",
```

Comment:
I'm on the fence if this is Info or Debug, thoughts?

Reply:
Might be good to clarify this is the control host though

Reply:
Done